book/website/reproducible-research/overview/overview-resources.md

```diff
@@ -21,6 +21,30 @@ If you want to start learning hands-on practices, we recommend reading the {ref}
 (rr-overview-resources-addmaterial)=
 ## Additional Material
 
+(rr-overview-resources-commandline)=
+### Using the command line
+
+While most of what this book is about do not require the use of the command line, it often makes things actually easier. 
```

Comment:
```suggestion
While most of what this book is about does not require the use of the command line, it often actually makes things easier. 
```
Could we expand here what "things" is referring to? Just in case someone completely new to the terminal comes across this page and isn't sure